packages/react-app/src/components/manage/disclaimer.js

```diff
@@ -1,9 +1,16 @@
-import {InputBox} from "../index";
-import React from "react";
+import {InputBox, Purple} from "../index";
+import React, {useContext} from "react";
+import {Context} from '@project/react-app/src/services'
 
 
 export default function Disclaimer(props) {
 
+    const context = useContext(Context)
+
+    const go = () => {
+        context.modals.triggerModal({message: "How To", component: "WatchVideo"})
```

Comment:
Where is this components definition and video URL?

Reply:
Heh.  Oops!  Let me find those 🥲